lib/mix/tasks/igniter.upgrade_igniter.ex

```diff
@@ -42,7 +42,8 @@ defmodule Mix.Tasks.Igniter.UpgradeIgniter do
     upgrades =
       %{
         "0.3.66" => [&code_module_parse_to_project_module_parse/2],
-        "0.3.71" => [&code_module_parse_to_project_module_parse/2]
+        "0.3.71" => [&code_module_parse_to_project_module_parse/2],
+        "0.3.76" => [&code_common_nth_right_to_move_right/2]
```

Comment:
I chose the version that I expect this to release in; is that the correct thing to do? (We're on 0.3.75 as of now.)

Reply:
Yep!